wiki/Tournaments/CFC/2018/en.md

```diff
@@ -0,0 +1,235 @@
+
+- NoMod
+  - [Getty vs. DJ DiA - Ops:Code-Rapture- \[GRAVITY\]](https://osu.ppy.sh/beatmapsets/562060/#fruits/1188198)
+  - [Silentroom - Nhelv \[Catastrophe\]](https://osu.ppy.sh/beatmapsets/684268/#fruits/1448121)
+  - [Camellia - gigadelic (Camellia's "The TERA" RMX) \[Crystal's Nervous Breakdown\]](https://osu.ppy.sh/beatmapsets/388122/#fruits/861028)
+  - [Omoi - Teo \[Motto!\]](https://osu.ppy.sh/beatmapsets/721804/#fruits/1524103)
+  - [rerulili - MA'JI'KI'TI'GIRL \[No One But CRN\]](https://osu.ppy.sh/beatmapsets/65660/#fruits/194320)
+  - [Sota Fujimori - polygon \[Incrementally Hi-Speed Extreme\]](https://osu.ppy.sh/beatmapsets/559097/#fruits/1234400)
+- Hidden
+  - [Various Artists - Dan ~ CTB ~ TechMap Pack \[Forgotten ~ Aleph ~ (Marathon)\]](https://osu.ppy.sh/beatmapsets/693829/#fruits/1468082)
+  - [USAO - Boss Rush \[MAXIMUM\]](https://osu.ppy.sh/beatmapsets/602460/#fruits/1272534)
+  - [Susumu Hirasawa - The Secret of The Flowers of Phenomenon \[Insane\]](https://osu.ppy.sh/beatmapsets/70247/#fruits/201974)
+  - [sana - Packet Hero \[Ruthless\]](https://osu.ppy.sh/beatmapsets/404910/#fruits/880321)
+- Hard Rock
+  - [Eleharmonica remixed by kors k - Der Wald (kors k Remix) \[Spec's Overdose\]](https://osu.ppy.sh/beatmapsets/581787/#fruits/1536964)
+  - [Memme - Avalanche \[Overdose\]](https://osu.ppy.sh/beatmapsets/357244/#fruits/785892)
+  - [xi - ANiMA \[Oberum Primus\]](https://osu.ppy.sh/beatmapsets/441271/#fruits/949234)
+  - [BLUE ENCOUNT - HOPE \[SURVIVOR\]](https://osu.ppy.sh/beatmapsets/685622/#fruits/1451282)
+- Double Time
+  - [Seiryu - Ultramarine \[Rain\]](https://osu.ppy.sh/beatmapsets/275991/#fruits/625941)
+  - [GRANRODEO - Can Do (TV Size) \[Platter\]](https://osu.ppy.sh/beatmapsets/580381/#fruits/1228853)
+  - [Hatsune Miku & Megpoid Gumi - Ashurashurashura \[Insane\]](https://osu.ppy.sh/beatmapsets/36248/#fruits/148911)
+  - [Hardwell & Joey Dale feat. Luciana - Arcadia \[Paradise\]](https://osu.ppy.sh/beatmapsets/258193/#fruits/590695)
+- FreeMod
+  - [Various Artists - Dan ~ CTB ~ JumpMap Pack \[Los! Los! Los! ~ Aleph ~ (Marathon)\]](https://osu.ppy.sh/beatmapsets/692137/#fruits/1464599)
+  - [Within Temptation - In The Middle Of The Night (Nightcore Mix) \[Nervous Breakdown\]](https://osu.ppy.sh/beatmapsets/603419/#fruits/1274497)
+  - [Renard - Rainbow Dash Likes Girls (Stay Gay Pony Girl) \[Holy Shit! It's Rainbow Dash!!\]](https://osu.ppy.sh/beatmapsets/57255/#fruits/172662)
+  - [Memme - Extreme Fantasy \[ExtrA\]](https://osu.ppy.sh/beatmapsets/49459/#fruits/152480)
+- Tiebreaker 
+  - [Camellia - Feelin Sky (Camellia's "200step" Self-remix) \[Overdrive\]](https://osu.ppy.sh/beatmapsets/713891/#fruits/1508761)
+  
+### Winner's Bracket + Loser's Bracket Finals
+
+**[Download the mappack here! (116.2MB)](https://mega.nz/#!R9VxiBoS!prokzQthV3M32505pt7dHmeLMejvV2U2dUhGegnhfHM)**
+
+- NoMod
+ 	- [BlackYooh vs. siromaru - BLACK or WHITE? \[EXTRA\]](https://osu.ppy.sh/beatmapsets/508846/#fruits/1082169)
```

Comment:
space?

Reply:
don't really understannd the issue with this one